frontend/public/components/overview/index.tsx

```diff
@@ -955,115 +981,127 @@ const overviewDispatchToProps = (dispatch) => {
   };
 };
 
-const Overview_: React.SFC<OverviewProps> = (({mock, namespace, selectedItem, selectedView, title, dismissDetails}) => {
-  const sidebarOpen = !_.isEmpty(selectedItem) && selectedView !== View.Dashboard;
-  const className = classnames('overview', {'overview--sidebar-shown': sidebarOpen});
-  // TODO: Update resources for native Kubernetes clusters.
-  const resources = [
-    {
-      isList: true,
-      kind: 'Build',
-      namespace,
-      prop: 'builds',
-    },
-    {
-      isList: true,
-      kind: 'BuildConfig',
-      namespace,
-      prop: 'buildConfigs',
-    },
-    {
-      isList: true,
-      kind: 'DaemonSet',
-      namespace,
-      prop: 'daemonSets',
-    },
-    {
-      isList: true,
-      kind: 'Deployment',
-      namespace,
-      prop: 'deployments',
-    },
-    {
-      isList: true,
-      kind: 'DeploymentConfig',
-      namespace,
-      prop: 'deploymentConfigs',
-    },
-    {
-      isList: true,
-      kind: 'Pod',
-      namespace,
-      prop: 'pods',
-    },
-    {
-      isList: false,
-      kind: 'Project',
-      name: namespace,
-      prop: 'project',
-    },
-    {
-      isList: true,
-      kind: 'ReplicaSet',
-      namespace,
-      prop: 'replicaSets',
-    },
-    {
-      isList: true,
-      kind: 'ReplicationController',
-      namespace,
-      prop: 'replicationControllers',
-    },
-    {
-      isList: true,
-      kind: 'Route',
-      namespace,
-      prop: 'routes',
-    },
-    {
-      isList: true,
-      kind: 'Service',
-      namespace,
-      prop: 'services',
-    },
-    {
-      isList: true,
-      kind: 'StatefulSet',
-      namespace,
-      prop: 'statefulSets',
-    },
-  ];
+const Overview_ = connectToFlags<OverviewProps>(FLAGS.KUBEVIRT)(
```

Comment:
I would avoid double-connecting the component and add the flag handling to `overviewStateToProps`. Here's an example:

https://github.com/spadgett/console/blob/a2344bbadd5c9f41a46285bf889eaa871dbf9ad3/frontend/public/components/namespace.jsx#L242-L247

Reply:
Will do, PR update coming in a few mins.